pypeit/spectrographs/keck_lris.py

```diff
@@ -688,6 +728,24 @@ def get_detector_par(self, hdu, det):
             ronoise=np.atleast_1d([4.54, 4.62])
         ))
 
+        # Allow for post COVID detector issues
+        t2020_1 = time.Time("2020-06-30", format='isot')  # First run
+        t2020_2 = time.Time("2020-07-29", format='isot')  # Second run
+        date = time.Time(hdu[0].header['MJD-OBS'], format='mjd')
+
+        if date < t2020_1:
+            pass
+        elif date > t2020_2:  # This is the 2020 July 29 run
```

Comment:
Am a bit scared about hard-coding the gains/ rdnoise.

Reply:
May wish to warn the user

Reply:
As an added comment, I think it would be better to turn the logic around, so that it's time-ordered (makes it easier to add new dates in the future):
```
        if date < t2020_1:
            pass
        elif date < t2020_2:
            # This is for the June 30 2020 run
            detector_dict1['gain'] = np.atleast_1d([37.6])
            detector_dict2['gain'] = np.atleast_1d([1.26])
            detector_dict1['ronoise'] = np.atleast_1d([99.])
            detector_dict2['ronoise'] = np.atleast_1d([5.2])
        else:
            # All future dates
            detector_dict1['gain'] = np.atleast_1d([1.45])
            detector_dict2['gain'] = np.atleast_1d([1.25])
            detector_dict1['ronoise'] = np.atleast_1d([4.47])
            detector_dict2['ronoise'] = np.atleast_1d([4.75])
```

Reply:
good suggestion done.  Warning added.